src/components/action-button.js

```diff
@@ -26,15 +30,20 @@ export const renderButton = ({ layout, theme, app, constraints, senseNavigation,
       const { actions } = layout;
       actions.forEach((action) => {
         const actionObj = allActions.find((act) => act.value === action.actionType);
-        actionObj && actionCallList.push(actionObj.getActionCall({ app, qStateName, ...action }));
+        if (actionObj) {
+          if (actionObj.group === 'dynamicViews') {
+            actionCallList.push(actionObj.getActionCall({ app, senseNavigation }));
+          } else {
+            actionCallList.push(actionObj.getActionCall({ app, qStateName, ...action }));
+          }
+        }
```

Comment:
No need for splitting here. Just add senseNavigation as input. The function can decide what to use from the input

Reply:
Done.